docs/userguide.md

```diff
@@ -712,3 +713,15 @@ A and B are offline:
 ```
 Replica layout: C=20
 ```
+
+## Controller-Manager Leader Election
+
+The federation controller manager is always deployed with leader election feature
+to ensure high availability of the control plane. Leader election module ensures
+there is always a leader elected among multiple instances which takes care of
+running the controllers. In case an active instance goes down, one of the standby
+gets elected as leader, ensuring minimum downtime. Leader election is necessary so that
+only one instance is responsible for reconciliation. You can refer to
+[helm chart configuration](https://github.com/kubernetes-sigs/federation-v2/tree/master/charts/federation-v2#configuration)
+to configure parameters for leader election module to tune to your environment
```

Comment:
s/module to tune to/to tune for/
s/environment although the defaults are sane/environment (the defaults should be...)/